python/edgar_filing_searcher/parsers/daily_index_crawler.py

```diff
@@ -0,0 +1,96 @@
+# pylint: disable=no-member
+"""This file crawls the daily index directory pages"""
+import logging
+import re
+import time
+from datetime import date, timedelta
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+from edgar_filing_searcher.errors import BadWebPageResponseException, InvalidUrlException
+
+
+def get_request_response(url):
+    """Returns the response request from the URL"""
+    backoff_factor_value = 3

```

Comment:
sometimes people will make these variables global or in another module so that they can be modified. This can be useful when you want to modify the values for tests.